package.json

```diff
@@ -47,22 +44,15 @@
     "react-flip-move": "3.0.5",
     "react-markdown": "8.0.4",
     "react-router-dom": "6.6.1",
-    "react-slick": "0.29.0",
```

Comment:
don't we need this for poll?

Reply:
I checked it and for the results it works (I guess because a4 installs it). Or is there another location where we use it and I am not aware of?